pandas/core/arrays/numeric.py

```diff
@@ -150,7 +150,7 @@ def _coerce_to_data_and_mask(
     if dtype is not None:
         dtype = dtype_cls._standardize_dtype(dtype)
 
-    cls = dtype_cls.construct_array_type()
+    cls = dtype_cls().construct_array_type()
```

Comment:
Nit: Is it possible to change `_coerce_to_data_and_mask` to pass an dtype instance here?

Reply:
thats slightly more invasive since it means updating the annotations etc

Reply:
OK good for a follow up then. Would like to avoid always assuming here that it's safe to instantiate these types with no arguments